rusk-recovery/src/keys.rs

```diff
@@ -47,6 +56,16 @@ static PUB_PARAMS: Lazy<PublicParameters> = Lazy::new(|| {
     }
 });
 
+#[tokio::main]
+async fn fetch_pp() -> Result<Vec<u8>, Box<dyn std::error::Error>> {
+    let response = reqwest::get("https://dusk-infra.ams3.digitaloceanspaces.com/trusted-setup/dusk-trusted-setup")
+        .await?
+        .bytes()
+        .await?;
+
+    Ok(response.to_vec())
+}
```

Comment:
Please take this as parameter, taking it as cli argument in rusk 

https://github.com/dusk-network/rusk/blob/c02ab74d2e5adc5b3f10da80799b2a42f74feb34/rusk/src/bin/args/command.rs#L13-L25